main.py

```diff
@@ -105,7 +112,8 @@ def __init__(self):
 		self._p = None
 		self._qmp = None
 
-	def start(self, settings):
+	@staticmethod
+	def generateLaunchCmd(settings):
```

Comment:
If an exception occurs here (which can be generated by check_path), the settings dialog will never open.

Reply:
Hm, my first thought would be to add a new parameter ("checkPaths" or something like that) to make path checking optional, so that we can skip this in the settings dialog but keep it for when we actually want to run xqemu.

Reply:
@thrimbor I'll trust you to come up with a good solution! Will merge once fixed.